src/db/table/user_payment.sql

```diff
@@ -0,0 +1,13 @@
+-- User payment records all users' payments on the platform
+CREATE TABLE `user_payment` (
+  `id` int(11) NOT NULL AUTO_INCREMENT,
+  `payment_type_id` int(11) DEFAULT NULL COMMENT 'Can be null if we decide to stop supporting a payment method',
```

Comment:
Maybe wiser to leave this as `NOT NULL` and have a column in `payment_type` called `enabled`? I'm looking to minimize destruction of data, even if it is legacy.